pkg/operator2/operator.go

```diff
@@ -41,13 +41,15 @@ import (
 var deploymentVersionHashKey = operatorv1.GroupName + "/rvs-hash"
 
 const (
-	clusterOperatorName = "authentication"
-	targetName          = "openshift-authentication"
-	targetNameOperator  = "openshift-authentication-operator"
-	globalConfigName    = "cluster"
+	clusterOperatorName     = "authentication"
+	targetName              = "oauth-server"
+	targetNamespace         = "openshift-authentication"
+	targetNameOperator      = "authentication-operator"
+	targetNamespaceOperator = "authentication-operator"
```

Comment:
For any const that you change, create a new variable name to make it clear what is getting updated.

Also, this const value is wrong.